packages/curve-ui-kit/src/themes/design/2_theme.ts

```diff
@@ -5,13 +5,17 @@ const { plain, inverted } = SurfacesAndText
 
 const Transition = `color ${TransitionFunction}, background-color ${TransitionFunction}, filter ${TransitionFunction}`
 
+const InsetOverline = '0 0 auto' as const // Top border only
```

Comment:
I don't understand these :thinking: 
`0 0 auto` means `{top: 0, right: 0, bottom: 'auto', left: 0}`, how is that top border only?

Reply:
'auto' is simply the default CSS value in order to be able to use the 4 value syntax of top-right-bottom-left.

`0 0 auto` is equivalent to `top: 0; right: 0` which means on top from left to right; so top border.
`auto 0 0` is equivalent to `bottom: 0; right: 0` which means on bottom from left to right; so bottom border.

Reply:
ah thx, tbh I never used inset :fearful: